sqlflow/client.py

```diff
@@ -172,7 +172,8 @@ def read_fetch_response(self, job_id):
                     def _rows_gen():
                         for row in rows:
                             yield [self._decode_any(d) for d in row.data]
-                    compound_message.add_rows(Rows(column_names, _rows_gen), None)
+                    if rows:
+                        compound_message.add_rows(Rows(column_names, _rows_gen), None)
                     rows = []
```

Comment:
Why do we have to set `rows` to `[]` here?

Reply:
There would be multiple SQL statements here, so we should reset the `rows` at the end of each statement execution.